db/migrate/20151015160334_change_author_and_text_to_not_null.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAuthorAndTextToNotNull < ActiveRecord::Migration
+  def change
+    change_column_null(:comments, :author, false, "")
+    change_column_null(:comments, :text, false, "")
+  end
+end
```

Comment:
Why 2 migrations?

Does change_column_null handle migrating old records? We might need a couple lines of SQL to update the existing records. Then again, there probably aren't any, or else the UI would have crashed.